src/Illuminate/Container/Container.php

```diff
@@ -706,7 +706,7 @@ public function get(string $id)
                 throw $e;
             }
 
-            throw new EntryNotFoundException($id, $e->getCode(), $e);
+            throw new EntryNotFoundException($id, is_int($e->getCode()) ? $e->getCode() : 0, $e);
```

Comment:
Couldn't you assign the result of the function call to save calling it twice when it is an `int`?

```suggestion
            throw new EntryNotFoundException($id, is_int($code = $e->getCode()) ? $code : 0, $e);
```

Appreciate this is a _very_ minor optimisation in this case.